Phase04/SimpleCalculator.Test/SimpleCalculator.Business.Tests.cs

```diff
@@ -0,0 +1,29 @@
+using SimpleCalculator.Business;
+using SimpleCalculator.Business.Enums;
+using FluentAssertions;
+
+namespace TestProject1;
+
+public class CalculatorBusinessTests
+{
+    [Theory]
+    [InlineData(10, 20, 30, OperatorEnum.sum)]
+    [InlineData(10, -10, 0, OperatorEnum.sum)]
+    [InlineData(50, 20, 30, OperatorEnum.sub)]
+    [InlineData(10, 20, 200, OperatorEnum.multiply)]
+    [InlineData(10, 0, 0, OperatorEnum.multiply)]
+    [InlineData(100, 10, 10, OperatorEnum.division)]
+    public void Calculate_AllOperators_TrueResultValue(int first, int second, int result, OperatorEnum operatorEnum)
+    {
+        var calculator = new Calculator(); 
+        var expected = calculator.Calculate(first, second, operatorEnum);
+        expected.Should().Be(result);
```

Comment:
Testing the `Calculate` function without mocking `IOperator` and `IOperatorProvider` violates Unit-Test principle.
`OperatorProvider` and each `Operator` class should be tested separately.
`Calculator` class should be tested with mocked implementations of the required interfaces.